\paragraph{Задание 6} \hspace{0pt}

\textbf{Условие}:
Напишите программу на С++, которая показывает, что любое исключение перехватывается с помощью catch (...).

\textbf{Решение}:

\lstinputlisting[language=C++, name=main.cpp,]
{../../src/task6/src/main.cpp}

\begin{lstlisting}[language=Out,]
= = = = = start try = = = = =
arcsin(0.5) = -0.30103
arcsin(-55) = catch(...)
\end{lstlisting}